### PR TITLE
Update mac dylib to libsndfile-1.0.30

### PR DESCRIPTION
**Pull request by tchagnon**
Addresses #3

### CODE REVIEW

**Comment by tchagnon** [comment]
Let me know if you want me to split the README / script update to a separate PR, etc.

**Comment by tchagnon** [comment]
Also, I confirmed that the built `.dylib` shouldn't have any non-system dependencies:
```
otool -L libsndfile.dylib                        
libsndfile.dylib:
        /usr/local/lib/libsndfile.1.dylib (compatibility version 2.0.0, current version 2.30.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1281.100.1)
```

**Comment by bastibe** [comment]
Thank you very much!

**Comment by mgeier** [comment]
The file `libsndfile.dylib` is suspiciously small (491k compared to 1.9M before). Did anyone test whether it actually works (and contains all static libraries that it claims to contain)?

File: README.md [hunk 1]
```diff
@@ -38,62 +38,18 @@ The XCode CLI tools were installed with:
...
-
-    cp -i $SNDFILENAME/src/.libs/libsndfile.1.dylib libsndfile.dylib
-    chmod -x libsndfile.dylib
+Edit the versions of required libraries in the `mac_build.sh` file
 
+    OGGNAME=libogg-1.3.4
+    VORBISNAME=libvorbis-1.3.7
+    FLACNAME=flac-1.3.3
+    OPUSNAME=opus-1.3.1
+    SNDFILE_VERSION=1.0.30
```
Comment (mgeier) [inline README.md:47]: Those numbers should only be stored in one place, i.e. only in the script.

Comment (tchagnon) [reply README.md:47]: ok, i can remove these in another pr

**Comment by tchagnon** [comment]
> The file `libsndfile.dylib` is suspiciously small (491k compared to 1.9M before). Did anyone test whether it actually works (and contains all static libraries that it claims to contain)?

Hmm. Sorry, no I didn't really test the dylib yet (though i had built from source before and tested).  I can try that today.

**Pull request merged**